frontend/src/scenes/insights/utils/dataStorage.ts

```diff
@@ -0,0 +1,28 @@
+import { InsightDataThatVariesWithContext, InsightModel } from '~/types'
```

Comment:
If it's a file that contains just one function, let's call the file the same as the function. Also, what's "storage" *exactly* in this context? It's always safer to use less vocabulary than more :).